packages/api/package.json

```diff
@@ -22,7 +22,7 @@
     "serve:debug": "LOCALSTACK_HOST=localhost LOCAL_ES_HOST=localhost NODE_ENV=test FAKE_AUTH=true node --inspect-brk ./bin/cli.js serve",
     "tsc": "../../node_modules/.bin/tsc",
     "tsc:listEmittedFiles": "../../node_modules/.bin/tsc --listEmittedFiles",
-    "test": "../../node_modules/.bin/ava",
+    "test": "echo $PWD && for i in `find tests/ -name \\*.js -exec grep -l require\\(\\'ava\\'\\) {} \\;`; do echo TEST START: $i && ../../node_modules/.bin/ava $i; done;",
```

Comment:
So what is this change meant to capture? Were we missing some tests? Does it have something to do with the ordering we're messing with in the --streamed logs?

Reply:
It's intended to capture concretely which test AVA is running at a point in time.

Reply:
Ah ok so the task failed, the logs are truncated/insufficient and this is meant to tell us which test we're on, ideally around the time the task fails.

Yeah, hopefully redundant but I see it in the logs and it seems potentially helpful.